ansible_collections/arista/avd/roles/eos_designs/doc/connected-endpoints.md

```diff
@@ -144,10 +146,6 @@ port_profiles:
         # Custom structured config added under ethernet_interfaces.<interface> for eos_cli_config_gen
         structured_config: < dictionary >
 
-      < port_profile_2 >:
```

Comment:
This is the main typo that I found. A dictionary in the middle of a list. Looks like a misplaced port_profile in adapters.